shallow_backup.py

```diff
@@ -156,18 +159,22 @@ def backup_dotfiles(backup_path):
 
 	# get dotfolders and dotfiles
 
-	dotfiles_for_backup = [".bashrc", ".bash_profile", ".gitconfig", ".pypirc", ".shallow-backup", ".zshrc"]
+	dotfiles_for_backup = [".bashrc", ".bash_profile",
```

Comment:
A great improvement that just came to mind would be extracting these types of lists to a constants file. I'll open an issue for this. (#67)

Reply:
Yea, right. I also noticed that the `.vimrc` could easily be added, doesn't it?

Reply:
Yep, let's do it.